### ISSUE TITLE
Namespace

### ISSUE THREAD

**Issue by frederikbosch** [RustJason/php-rapidjson#1]
Great work already! I wish I could help but lacking C skills and knowledge of PHP C codebase. I know it is work in progress, but why not create a namespace for Rapidjson? There are releases of Rapidjson planned that [have more features](https://github.com/miloyip/rapidjson/issues/454). You might need more objects then. Having namespaces already would help when adding more features.

**Comment by rustjson** [comment]
Did you mean this?

``` php
<?php
use rapidjson;

new parser();
```

**Comment by frederikbosch** [comment]
Yes, something like that. I think that would be more up to current standards.

**Comment by rustjson** [comment]
Ok, I will try

**Comment by ethanhann** [comment]
For namespaces, as it is you can do...

``` php
use Rapidjson;

new Rapidjson();
```

Or without the use operator...

``` php
new \Rapidjson();
```
